packages/cli/src/databases/repositories/test-case-execution.repository.ee.ts

```diff
@@ -0,0 +1,88 @@
+import { Service } from '@n8n/di';
+import { DataSource, In, Not, Repository } from '@n8n/typeorm';
+import type { DeepPartial } from '@n8n/typeorm/common/DeepPartial';
+
+import { TestCaseExecution } from '@/databases/entities/test-case-execution.ee';
+
+@Service()
+export class TestCaseExecutionRepository extends Repository<TestCaseExecution> {
+	constructor(dataSource: DataSource) {
+		super(TestCaseExecution, dataSource.manager);
+	}
+
+	async createBatch(testRunId: string, pastExecutionIds: string[]) {
```

Comment:
This can be a bit misleading name, as in typeorm `create` refers to instantiating an instance of the entity. Here we also insert to the database. Maybe it could be `insertBatch` or `saveBatch` or `*Many`